google/dm-solutions/README.md

```diff
@@ -0,0 +1,6 @@
+# Deployment Manager Solutions
+
+This directory contains Google created and managed Deployment Manager solutions. Deployment Manager team will frequently test all solutions under "dm-solutions" folder.
```

Comment:
Rephrase to "This directory contains Deployment Manager solutions that are created and managed by Google. The Deployment Manager team will test and maintain these samples."